src/components/BottomNav.vue

```diff
@@ -0,0 +1,57 @@
+<template>
+  <div>
+    <div>Active: {{ bottomNav }} </div>
+    <div>Coach: {{ coach }} </div>
+    <div>Manager: {{ manager }} </div>
+    <div>Admin: {{ admin }} </div>
+    <v-bottom-nav
+      :active.sync="bottomNav"
+      :value="true"
+      fixed
```

Comment:
```suggestion
      app
```
Should use app not fixed